app/views/layouts/overdue.html.erb

```diff
@@ -79,19 +79,20 @@
         <% end %>
       </section>
 
-      <section>
+      <section class="mt-5">
         <% if current_facility %>
           <% if @patient_summaries.present? && policy([:overdue_list, @patient_summaries.first]).download? %>
+            <h4>Download</h4>
             <%= link_to(@index_params.merge(format: "csv"), class: "", data: { confirm: I18n.t('admin.phi_download_alert') }) do %>
-              <h4>Download Results</h4>
```

Comment:
We lost the anchor text here with this change. 😁 I'll create a PR to get it back.